### DISCUSSION TITLE
Does airflow user can be a non-root group user?

### DISCUSSION THREAD

**Discussion by yanghongtian** [apache/airflow#37126] [Q&A]
I want to deploy my airflow with a custom airflow user, for example, whose GID != 0 and UID = 1000, any ideas?

thanks!

**Comment by Taragolis** [comment]
I assume that you talk about Airflow under official docker image, if so.. then you can change the UID to any UID, and you can't change the GID. For more details have a look: [Allowing arbitrary user to run the container](https://airflow.apache.org/docs/docker-stack/entrypoint.html#allowing-arbitrary-user-to-run-the-container)
